### PR TITLE
go-tarantool: update to v2

### LINKED ISSUE #6: api: use go-tarantool/v2
We need to update version of the `go-tarantool` from v1 to v2 to be able to use TLS password/TLS password file for a connection.

### PR DESCRIPTION
**Pull request by DerekBum**
This patch updates go-tarantool version used to v2.

Closes #6

### CODE REVIEW

File: example_test.go [hunk 1]
```diff
@@ -264,22 +273,34 @@ func ExampleMap_insertMappedTuples() {
 			return
 		}
 		insertReq := tarantool.NewInsertRequest("test_space").Tuple(mapped)
-		resp, err := conn.Do(insertReq).Get()
+		resp, err := conn.Do(insertReq).GetResponse()
```
Comment (oleg-jukovec) [inline example_test.go]: Here I didn't quite understand why not `Get` as before?

File: example_test.go [hunk 2]
```diff
@@ -264,22 +273,34 @@ func ExampleMap_insertMappedTuples() {
...
+			return
+		}
+		if resp.Header().Error != tarantool.ErrorNo {
+			fmt.Println("insert response error =", resp.Header().Error)
+		}
 	}
 
 	selectReq := tarantool.NewSelectRequest("test_space")
-	resp, err := conn.Do(selectReq).Get()
+	resp, err := conn.Do(selectReq).GetResponse()
```
Comment (oleg-jukovec) [inline example_test.go]: The same.

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Changed
+- bump `go-tarantool` dependency to `v2` (#6).
```
Comment (oleg-jukovec) [inline CHANGELOG.md]: Please, fix all entries in the file if there are the same mistake.
```suggestion
- Bump `go-tarantool` dependency to `v2` (#6).
```

Comment (oleg-jukovec) [inline CHANGELOG.md]: The same for the entire file. See: http://keepachangelog.com/en/1.0.0/
```suggestion
### Changed

- bump `go-tarantool` dependency to `v2` (#6).
```

**Comment by DerekBum** [comment]
Updated.

**Review by oleg-jukovec — APPROVED** [review]
Thank you. It looks good to me, but let's try using this commit in the `tt-ee` before merge.

**Comment by DerekBum** [comment]
> Thank you. It looks good to me, but let's try using this commit in the `tt-ee` before merge.

`tt-ee` PR with this version of go-tupleconv: https://github.com/tarantool/tt-ee/pull/237

**Pull request merged**